app/src/main/java/com/lastaoutdoor/lasta/viewmodel/DiscoveryScreenViewModel.kt

```diff
@@ -46,4 +49,25 @@ constructor(private val repository: OutdoorActivityRepository) : ViewModel() {
               node.tags.name ?: "Unnamed Climbing Activity"))
     }
   }
+  /*Toggles the dialog box on/off*/
+  val displayDialog = mutableStateOf(false)
+  /*Just a default activity to fill in the mutable state*/
+  val dummyActivity = OutdoorActivity(ActivityType.HIKING, 3, 5.0f, "2 hours", "Zurich")
+  val activityToDisplay = mutableStateOf(dummyActivity)
+
+  @Composable
```

Comment:
Should not have composable in viewmodel. This is bad practice

Reply:
Done